modules/apps/portal/portal-tika/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
 	compileInclude group: "com.liferay", name: "poi", version: "5.2.2.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "poi-ooxml", version: "5.2.2.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "poi-ooxml-lite", version: "5.2.2.LIFERAY-PATCHED-1"
+	compileInclude group: "com.liferay", name: "poi-scratchpad", version: "5.2.2.LIFERAY-PATCHED-1"
 	compileInclude group: "com.liferay", name: "xmlbeans", version: "5.0.3.LIFERAY-PATCHED-1"
 	compileInclude group: "com.pff", name: "java-libpst", version: "0.8.1"
 	compileInclude group: "com.uwyn", name: "jhighlight", version: "1.0"
```

Comment:
@javalosjr96, please add it to source-formatter.properties (place the change to the commit https://github.com/liferay-core-infra/liferay-portal/pull/2352/commits/c46b65b919bc52a82a7df732a7b18617e3b6a274). And then you may use "Auto SF" to generate. Since I noticed tika-parsers also existed in the source-formatter.properties and only one module used it.